modules/basics/src/test/java/com/opengamma/basics/date/HolidayCalendarTest.java

```diff
@@ -483,6 +486,37 @@ public void test_previousOrSame_null() {
   }
 
   //-------------------------------------------------------------------------
+  @DataProvider(name = "nextLastOrSameInMonth")
+  static Object[][] data_nextLastOrSameInMonth() {
+    return new Object[][] {
+        {THU_2014_07_10, THU_2014_07_10},
```

Comment:
Maybe the cal should be a test input as well (like ImmutableHolidayCalendarTest)